docs/courses/csintro1/loops/projectiles.md

```diff
@@ -83,11 +83,14 @@ projectile = sprites.createProjectile(img`
 `, 50, 0, SpriteKind.Player)
 ```
 
-This modified version adds in an event that triggers when the projectile is destroyed - you should see that when the sprite goes off the screen, it is actually automatically destroyed! 
+This modified version adds in an event that triggers when the projectile is destroyed - you should see that when the sprite goes off the screen, it is actually destroyed automatically! 
 
 ``||sprites:auto destroy||`` is available for all sprites using the ``||sprites:set sprite auto destroy on||`` flag, but projectiles have it added in for you automatically. 
 
-If projectiles are not provided a name of a sprite to start from (by pressing the ``+`` button on the block to show the option), they will be created either in the middle of the screen if the velocities in the x and y direction are both set to 0, or from the side of the screen opposite their initial speed otherwise (so that they can move across the screen).
+If projectiles are not set to start from another sprite (there's another a projectile block with a place for the start sprite name), they are created and will start from one of two positions:
+
+* In the middle of the screen if the velocities in the x and y direction are both set to `0`
```

Comment:
This got changed by projectile changes (https://github.com/Microsoft/pxt-common-packages/pull/556), but I'm going through and updating projectiles already so nothing to change here

Reply:
Oh yea, do you want to change the "projectile from" section to use **sprites.createProjectileFromSprite**?

Reply:
Sorry, you answered that just above.

Reply:
Yeah all references will end up being changed as old projectile is deprecated~